tests/test.ts

```diff
@@ -142,6 +143,42 @@ describe.skip('Test clasp deploy function', () => {
   });
 });
 
+describe.skip('Test clasp version and versions function', () => {
+  let versionNumber = '';
+  it('should create new version correctly', () => {
+    const result = spawnSync(
+      'clasp', ['version'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain('Created version ');
+    expect(result.status).to.equal(0);
+    versionNumber = result.stdout.substring(result.stdout.lastIndexOf(" "), result.stdout.length-2);
+    it('should list versions correctly', () => {
+      const result = spawnSync(
+        'clasp', ['versions'], { encoding: 'utf8' },
+      );
+      expect(result.stdout).to.contain('~ ');
```

Comment:
Ideally changing the `~` character (i.e. `–`) wouldn't break this test.
I suppose it's not that big of a deal if we change it later.